tests/Responses/Expected/product_attribute/show_response_after_partial_update.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": @integer@,
+    "code": "mug_material",
+    "type": "text",
+    "values": [],
+    "position": 0,
+    "createdAt": "@string@.isDateTime()",
+    "updatedAt": "@string@.isDateTime()",
+    "translations": {
+        "de": {
```

Comment:
I'm not sure if this is expected behavior: the test updates the attribute and writes new values for the `de_CH` and `en_US` translations. I'd expect it to replace existing translations, not add to the list. What do you think?